PoC/kafka-pull/README.md

```diff
@@ -1,41 +1,66 @@
+controller class, `KafkaBrokerDataFlowController`, manages the following operations:
 
-#### Properties
+1. **Start Transfer**: Creates OAuth credentials, generates tokens, and builds an Endpoint Data Reference (EDR) for the
+   consumer
+2. **Suspend/Terminate Transfer**: Revokes tokens and cleans up credentials
+3. **Token Management**: Integrates with OAuth services for secure authentication
 
-| Key                       | Description                                                                                                         | Mandatory |
-|:--------------------------|:--------------------------------------------------------------------------------------------------------------------|-----------|
-| `type`                    | Identifier of kafka data address. Should be set as 'KafkaBroker'                                                    | `true`    |
-| `topic`                   | Defines the Kafka topic                                                                                             | `true`    |
-| `kafka.bootstrap.servers` | Defines a custom endpoint URL to kafka                                                                              | `true`    |
-| `kafka.sasl.mechanism`    | Defines the SASL kafka mechanism (SCRAM-SHA-256 or SCRAM-SHA-512)                                                   | `true`    |
-| `kafka.security.protocol` | Defines the kafka  security protocol (SASL_PLAINTEXT or SASL_SSL)                                                   | `true`    |
-| `pollDuration`            | Defines the duration of the consumer polling. The value should be a ISO-8601 duration e.g. "PT10S" for 10 seconds.  | `false`   |
-| `secretKey`               | Defines the `vault` entry containing the secret token/credentials                                                   | `true`    |
-| `groupPrefix`             | Defines the groupPrefix that will be allowed to use for the consumer                                                | `true`    |
+## DataAddress Schema
 
-#### Secret Resolution
+When creating a Kafka asset, use the following properties in the DataAddress:
 
-The `secretKey` property should point to a `vault` entry that contains a string object with admin kafka credentials.
+| Key                       | Description                                                                                                        | Mandatory |
+|:--------------------------|:-------------------------------------------------------------------------------------------------------------------|-----------|
+| `type`                    | Identifier of kafka data address. Should be set as 'KafkaBroker'                                                   | `true`    |
+| `topic`                   | Defines the Kafka topic                                                                                            | `true`    |
```

Comment:
decide if Kafka is written with capital or small letter :-)

Reply:
done